connectors/src/connectors/github/temporal/workflows.ts

```diff
@@ -215,6 +215,8 @@ export async function githubRepoSyncWorkflow(
     nextCursor = cursor;
   }
 
+  // TOOD(spolu) add code syncing
```

Comment:
Will this be its own workflow ? do we get webhooks for every commit to main ?

Reply:
We call the activity from here so that we do sync code on permission change, but then we only sync code periodically (starting once a day). No webhooks for now.

Reply:
Ok got it 👍

Reply:
(The workflow integration is not done for now)